requirements_test.txt

```diff
@@ -39,6 +39,7 @@ types-chardet==0.1.5
 types-decorator==0.1.7
 types-enum34==0.1.8
 types-ipaddress==0.1.5
+types-paho-mqtt==1.6.0.1
```

Comment:
Are these new type stubs issed via the existing imports?

Reply:
> Are these new type stubs issed via the existing imports?

Yes.  Previously mypy would have just ignore / set to `Any` any imports from `paho.mqtt`. It will use the stubs now instead. https://mypy.readthedocs.io/en/stable/running_mypy.html#missing-library-stubs-or-py-typed-marker